packages/node-bridge/launcher.js

```diff
@@ -132,6 +142,54 @@ function getVercelLauncher({
   };
 }
 
+/**
+ * @param {Function} listenerHandler
+ * @returns {(req: import('./types').VercelRequest, res: import('./types').VercelResponse) => void}
+ */
+function wrapListener(listenerHandler) {
+  // TODO: how can we best test the timeout and warning behavior?
+  return async function (req, res) {
+    enforceTimeout(res);
+    return await validateResponse(listenerHandler, req, res);
+  };
+}
+
+/**
+ * @param {Function} listenerHandler
+ * @param {import('./types').VercelRequest} req
+ * @param {import('./types').VercelResponse} res
+ */
+async function validateResponse(listenerHandler, req, res) {
+  const resp = await listenerHandler(req, res);
+  if (typeof resp !== 'undefined') {
+    // TODO: add vercel.link ?
```

Comment:
We could make a vercel.link (or use an existing one, if it exists) that points to https://vercel.com/docs/concepts/functions/serverless-functions

Reply:
Created https://vercel.link/serverless-function-quickstart